src/wwwroot/template-editor/adapterService.js

```diff
@@ -32,18 +32,59 @@
     };
     
     var langKey = null;
+    var loginSession = null;
     init();
 
     return service;
 
     function init() {
       langKey = getPreferredLanguage();
+      loginSession = getLoginSession('relayLogin');
     };
 
+    function getLoginSession(storageName) {
+      var login = angular.fromJson($window.localStorage.getItem(storageName));
+      if (!login) {
+        logOut();
+      }
+      return login;
+    }
+
     function getPreferredLanguage() {
       var key = $window.localStorage.getItem('lang');
       return key || 'en';
     };
+
+    function logOut() {
+      $window.location = "/";
+    }
+
+    function getAccountId() {
+      if (isTokenExpired()) {
+        logOut();
+      }
+      return loginSession.accountId;
+    }
+
+    function getAccountName() {
+      if (isTokenExpired()) {
```

Comment:
Please, in order to simplify remove all references to something not implemented.